src/pages/settings/Troubleshoot/TroubleshootPage.tsx

```diff
@@ -101,7 +108,7 @@ function TroubleshootPage({shouldStoreLogs}: TroubleshootPageProps) {
                         isCentralPane
                         subtitleMuted
                         illustration={LottieAnimations.Desk}
-                        illustrationStyle={[styles.mt4, styles.mbn4]}
+                        illustrationStyle={!isNativePlatform && [styles.mt4, styles.mbn4]}
```

Comment:
Hmm why are we making a platform-specific style here? Can't we find a solution that works everywhere?

Reply:
Agreed, Why this page specifically has this problem whereas other similar pages(about page) are just fine.